packages/gatsby-theme-newrelic/src/components/SwiftypeSearch.js

```diff
@@ -143,6 +154,7 @@ const InputView = ({ getAutocomplete, getInputProps }) => {
 
 SwiftypeSearch.propTypes = {
   className: PropTypes.string,
+  isOpen: PropTypes.bool,
```

Comment:
We can remove this now because we aren't using it!

Reply:
derp! thanks for catching that!